server/utilities/currentEnv.js

```diff
@@ -2,6 +2,11 @@
  * Gets the current environment.
  * @returns {string} environment
  */
-const getCurrentEnv = () => process.env.NODE_ENV || 'development';
+export const getCurrentEnv = () => process.env.NODE_ENV || 'development';
 
-export default getCurrentEnv;
+/**
+ * Gets the app url based on the current environment
+ */
+export const getUrl = getCurrentEnv() === 'production'
+  ? process.env.APP_URL
```

Comment:
APP_URL and LOCAL_URL are technically the same thing but the value should change based on the environment. We don't need to have seperate env variables

Reply:
@seyi-adeleke I'd fix this.